### ISSUE TITLE
enhancement : add video support 

### ISSUE THREAD

**Issue by ingyesid** [yazeed44/MultiImagePicker#10]
add support for pick videos

**Comment by yazeed44** [comment]
Thanks
Is this a personal need ?

**Comment by ingyesid** [comment]
yes, i am working in a mobile messenger application and i need a chooser for pictures and videos .

**Comment by yazeed44** [comment]
I can't promise you that i will implement that , since my highest priority is images right now

**Comment by matthewbjones** [comment]
@yazeed44 our project would like video support as well. Is there any progress on this? Would you be willing to accept a PR if we forked the project, added video support and sent up the PR for review?

**Comment by yazeed44** [comment]
@matthewbjones 
>Is there any progress on this?

Nope and I don't think there will since I'm going to university soon

>Would you be willing to accept a PR if we forked the project, added video support and sent up the PR for review?

Sure that would make me happy. I'd suggest you wait until i finish of implement [mvp](https://medium.com/@artem_zin/m-model-from-mvc-mvp-in-android-flow-and-mortar-bd1e50c45395) Since the code by then will be much cleaner and simpler
